admin/services/data-access/pupil-census-import.data.service.js

```diff
@@ -69,4 +77,25 @@ const bulkLoadData = (connection, pupilData, schools, jobId) => {
   })
 }
 
+/**
+ * Bulk upload Error handler
+ * @param {Object} error
+ * @return {String} - Error message
+ */
+const handleError = (error) => {
+  if (!error) {
+    return 'Error received is undefined'
+  }
+  if (typeof error !== 'object') {
+    return `Error type is ${typeof error} with value ${error.toString()}`
+  }
+  if (error || Object.keys(error).length === 0) {
+    return `Error with no properties: ${error.toString()}`
+  }
+  if (error || !error.message) {
+    return `Error with no message property: ${error.toString()} `
+  }
+  return error && error.message.toString()
```

Comment:
this is much better.  i would just modify the last line, and call toString on the error object, rather than message.  as we know message is always a string anyway.

Reply:
@GuyHarwood done